app/src/main/java/com/pixlee/pixleeandroidsdk/ui/BaseFragment.java

```diff
@@ -63,4 +77,142 @@ public void onClick(DialogInterface dialog, int which) {
         b.show();
 
     }
+
+
+    void callMediaPicker() {
+        Intent intent = new Intent(Intent.ACTION_PICK);
+        intent.setDataAndType(MediaStore.Images.Media.EXTERNAL_CONTENT_URI, "image/* video/*");
+        startActivityForResult(intent, REQ_MEDIA_PICKER);
+    }
+
+    private final int REQ_MEDIA_PICKER = 1314;
```

Comment:
what is this constant?

Reply:
This code is for our demo app and is used to communicate with the OS. This code below request the OS to launch an image picker, an application, built in the OS with an Integer and return the picture or video that the user select. the returned data can be filtered with the same integer.
```
Intent intent = new Intent(Intent.ACTION_PICK);
intent.setDataAndType(MediaStore.Images.Media.EXTERNAL_CONTENT_URI, "image/* video/*");
startActivityForResult(intent, REQ_MEDIA_PICKER);
```

The OS returns the selected content data via onActivityResult() which is used to receive data from internal and external applications.
```
@Override
    public void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);
        if (resultCode == Activity.RESULT_OK && data != null) {
            if (requestCode == REQ_MEDIA_PICKER) {
                extractImage(data);
            }
        }
    }
}
```

Reply:
might be worth adding a comment to explain the constant